### PR TITLE
wording tweaks to the website

### PR DESCRIPTION
**Pull request by amjokisaari**
Some copy editing, wording changes for clarity, added workshop VI.

### CODE REVIEW

File: benchmarks/index.md [hunk 1]
```diff
@@ -50,22 +50,22 @@ with complex geometries (e.g., dendritic growth) and topology changes,
 such as particles merging or splitting, is challenging. The other
 mesoscale modeling approach uses diffuse (finite width)
 interfaces. This method can easily track complex interface geometries
-and topological changes such as particles merging. This benefit is not
+and topological changes such as particle merging. This benefit is not
```
Comment (tkphd) [inline benchmarks/index.md]: I think "particles" is more grammatically correct, but I could be mistaken, or there may be a term-of-art involved?

File: benchmarks/index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ At the macroscale, on the other hand, materials properties are
 averaged over many domains (e.g., compositional, structural, magnetic,
 or ferroelectric domains). Central entities at the macroscale are
 usually materials properties that describe a response to some
-macroscopic external forces, for example conductivity, elastic
+macroscopic external forces, e.g., conductivity, elastic
```
Comment (tkphd) [inline benchmarks/index.md]: ```*e.g.*```, since *exempli gratia* is not an English phrase

File: submit_a_new_code.md [hunk 3]
```diff
@@ -8,11 +8,15 @@ comment: >-
...
-installation, use, and comprehension of the code.
+This website is intended to help users make their own decisions regarding
+the suitability of the available codes for their research goals. For example,
+all codes are not equally suited to for solving all types of physics that may
+be incorporated into phase field models.  Or, a certain trade-off between
+performance and flexibility may be needed.  The numerical performance of the codes
+for the various benchmark problems will likely form the major basis for
+comparison, but other factors to consider include the ease of installation,
+learning curve to use the code, software compatiblity with different computing
+platforms, available documentation, etc.
```
Comment (tkphd) [inline submit_a_new_code.md]: Please don't conclude with "etc.", since "factors to consider include" makes it implicit.

File: workshops.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
-benchmark problems and provide other feedback, such as information for
-developers of open-source codes on how easily their codes can be
-installed. Significant portions of the workshop are devoted to
+An integral component of the CHiMaD [phase field benchmarking effort]({{ site.baseurl }}/#benchmarks)
+is regular workshops and associated [hackathons]({{ site.baseurl
+}}/hackathons).  Approximately twice a year, a spectrum of researchers
+from across the United States and internationally, including both
+graduate students and senior researchers, come together to participate.
+Workshops help us formulate, test, and revise the benchmark problems and
+allow a valuable venue for other feedback to be provided, such as
```
Comment (tkphd) [inline workshops.md]: Not a fan of "allow a valuable venue," it feels draconian. Maybe "provide a collaborative atmosphere for brainstorming and constructive criticism," or something along those lines?

File: workshops.md [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
+information for developers of open-source codes on how easily their
+codes can be installed. Significant portions of the workshop are devoted to
 discussion, which provide a chance for the community to become more
 integrated and allow us to work together to determine future needs and
```
Comment (tkphd) [inline workshops.md]: Oh, I see where the "allow" came from. Whelp, this needs work.

File: benchmarks/index.md (hunk 2 above)
Comment (amjokisaari) [reply benchmarks/index.md]: the asterisks will add formatting?  I actually don't know how to preview the changes as a web page...

File: benchmarks/index.md (hunk 1 above)
Comment (amjokisaari) [reply benchmarks/index.md]: I've always seen it written as I changed it, but I am not enough of a grammarian to look up the rule quickly.  I will reword it to avoid entirely.

File: submit_a_new_code.md (hunk 3 above)
Comment (amjokisaari) [reply submit_a_new_code.md]: interesting thought.  I'll reword to avoid conflict on this.

File: workshops.md (hunk 5 above)
Comment (amjokisaari) [reply workshops.md]: I'll work on it.

File: benchmarks/index.md (hunk 2 above)
Comment (tkphd) [reply benchmarks/index.md]: Yes, the ```md``` extension is for markdown; use [GitHub flavored syntax](https://guides.github.com/features/mastering-markdown/). To preview the website, or check your formatting, I would create a [gist](https://gist.github.com) with your content and render it. For example, https://gist.github.com/tkphd/5b57a930d5b4ed46e0a9cac805cfa6d6

Comment (amjokisaari) [reply benchmarks/index.md]: oooh thanks.

Comment (stvdwtt) [reply benchmarks/index.md]: Also, if you happen to use Atom as your editor, there's a plugin to preview markdown content that I've found to be really helpful:
https://atom.io/packages/markdown-preview-plus

Comment (tkphd) [reply benchmarks/index.md]: Oh brilliant, thanks @stvdwtt!

**Comment by wd15** [comment]
@amjokisaari, merging this now, thanks so much.

**Comment by amjokisaari** [comment]
great :)
